src/renderer/lib/state.js

```diff
@@ -107,7 +107,8 @@ function setupSavedState (cb) {
       downloadPath: config.DEFAULT_DOWNLOAD_PATH,
       isFileHandler: false,
       openExternalPlayer: false,
-      externalPlayerPath: null
+      externalPlayerPath: null,
+      startup: true
```

Comment:
Let me be clear, I'm not against create a startup feature, but do not force the user beforehand.
* The application is not a daemon, so the user should indicate when start.
* The user will be surprised with the first autostart (even more if its a previous user).
* A lot of users care about their privacy, hence use VPNs, (manually started in most of the cases) so first autostart will be offensive. 
* Even in the forum post you mention, the users ask for the same expected behavior
![image](https://cloud.githubusercontent.com/assets/6145761/18027320/3c7d218a-6c37-11e6-9cb1-0439c225885f.png)
he is not complaining about not respecting user's preference but the default behavior.